specification/containerservice/resource-manager/readme.java.md

```diff
@@ -16,7 +16,20 @@ description: "Container Service Client"
 
 ``` yaml $(java) && $(multiapi)
 batch:
-  - tag: package-2019-06
```

Comment:
Should keep the original one.

Reply:
Does it need to generate all these two package? I think this tag is just used for generate sdk and don't need to generate the old one again.

Reply:
Done